Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabInstanceUtils.h

```diff
@@ -47,12 +51,18 @@ namespace AzToolsFramework
             //! @return The relative path string.
             AZStd::string GetRelativePathFromClimbedInstances(const AZStd::vector<const Instance*>& climbedInstances);
 
+            //! Generates a relative path from reverse iterators of climbed instances.
+            //! @param climbedInstancesBeginIter The begin reverse iterator of climbed instances.
+            //! @param climbedInstancesBeginIter The end reverse iterator of climbed instances.
+            //! @return The relative path string.
+            AZStd::string GetRelativePathFromClimbedInstances(
```

Comment:
This way of overloading functions isn't a good API practice imo as this can lead to confusion on which version to call. Is there a reason the previous function cannot be reused here?

Reply:
According to the comment In PrefabFocusHandler::AppendPathFromFocusedInstanceToPatchPaths, we call GetRelativePathFromClimbedInstances(++climbUpResult.m_climbedInstances.crbegin(), climbUpResult.m_climbedInstances.crend()) since we want to skip the instance closest to the target instance since I don't want to particularly create a vector for climbedInstances without the skipped instance.